tests/foreman/ui/test_host.py

```diff
@@ -1,14 +1,278 @@
 # -*- encoding: utf-8 -*-
-from nailgun import entities
+from fauxfactory import gen_string
+from nailgun import entities, entity_mixins
+from robottelo.api.utils import promote
+from robottelo.config import conf
 from robottelo.constants import ENVIRONMENT
 from robottelo.decorators import run_only_on
 from robottelo.test import UITestCase
+from robottelo.ui.base import Base
 from robottelo.ui.locators import common_locators
 from robottelo.ui.session import Session
 
 
 @run_only_on('sat')
-class Host(UITestCase):
+class Host(UITestCase, Base):
+
+    hostname = gen_string('numeric')
+
+    @classmethod
+    def setUpClass(cls):  # noqa
```

Comment:
#noqa shouldn't be required here.  Which error are you trying to skip using this?